combat-trainer.lic

```diff
@@ -1155,8 +1155,8 @@ class SpellProcess
     @command = nil
   end
 
-  def check_spell_timer(data)
-    Time.now - (@spell_timers[data['abbrev']] || Time.at(0)) >= data['recast_every']
+  def check_spell_timer?(data)
+    Time.now - (@spell_timers[data['pet_type']] || @spell_timers[data['abbrev']] || Time.at(0)) >= data['recast_every']
```

Comment:
💡 Extracting `@spell_timers[data['pet_type']] || @spell_timers[data['abbrev']] || Time.at(0)` into a well-named variable would add a lot of readability to this function.